isthmus/src/main/java/io/substrait/isthmus/expression/LiteralConstructorConverter.java

```diff
@@ -25,13 +25,13 @@ public Optional<Expression> convert(
           ExpressionCreator.list(
               false,
               call.operands.stream()
-                  .map(t -> ((Expression.Literal) t.accept(new RexExpressionConverter())))
+                  .map(t -> ((Expression.Literal) topLevelConverter.apply(t)))
                   .toList()));
 
       case SqlMapValueConstructor map -> {
         List<Expression.Literal> literals =
             call.operands.stream()
-                .map(t -> ((Expression.Literal) t.accept(new RexExpressionConverter())))
+                .map(t -> ((Expression.Literal) topLevelConverter.apply(t)))
```

Comment:
Avoiding the use of `new RexExpressionConverter()`
This constructor doesn't setup all the callConverters.